packages/backend-plugin-api/api-report.md

```diff
@@ -354,6 +354,9 @@ export interface IdentityService extends IdentityApi {}
 
 export { isChildPath };
 
+// @public
+export function isDatabaseConflictError(e: unknown): boolean;
```

Comment:
Bit conflicted :hehe: about having this in `backend-plugin-api`, but we've already discussed that and there's no obvious better option atm. Let's move it but keep an eye out for options before 1.0

Reply:
Yes, there is another database helper, the `dropDatabase` function, but it seems coupled to the `DatabaseMananger` connectors. So I thought it made sense to keep it next to the DatabaseManager in the `backend-defaults` package. Do you think I should move it to the plugin API insteas?

Reply:
No I think that's good for now. Just trying to judge how much we need `@backstage/knex` or similar :p

Reply:
🙌🏻